### DISCUSSION TITLE
[제안] OpenCV를 이용한 이미지 이진화

### DISCUSSION THREAD

**Discussion by remaindere** [bcaitech1/p4-ocr-hansarang#19] [Ideas]
현재 저희 대회의 베이스라인 코드가 greyscale로 이미지를 읽어오는 것이 default 값으로 되어 있고, 특히나 바탕색과 확연히 구분되는 색상으로 필기를 해 놓은 이미지 특성 상 이진화를 사용해 보면 어떨까 생각됩니다.

다양한 이진화 방법에 대해 다뤄놓은 글을 소개합니다.
reference : https://jvvp.tistory.com/1083

저희 이미지의 조명값이 평균적으로 일정하고, 다양한 배경색이 사용되지 않으므로 Adaptive Threshold 방식의 이진화를 선택하여 사용해 보는것이 좋을 듯 합니다.

Original / Adaptive Threshold
![image](https://user-images.githubusercontent.com/48322490/120209715-f607e800-c269-11eb-9204-18ba965fc77b.png)

**Comment by jo-member** [comment]
wow 이진화 하면 잘나올수도있겠네요 👍
